src/main/scala/com.mediative.eigenflow/process/ProcessSupervisor.scala

```diff
@@ -0,0 +1,47 @@
+package com.mediative.eigenflow.process
+
+import java.util.Date
+
+import akka.actor.SupervisorStrategy.Stop
+import akka.actor.{ Actor, ActorRef, OneForOneStrategy, Props, SupervisorStrategy, Terminated }
+import com.mediative.eigenflow.StagedProcess
+import com.mediative.eigenflow.publisher.MessagingSystem
+
+import scala.util.{ Failure, Success, Try }
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object ProcessSupervisor {
+  def props(process: StagedProcess, date: Option[Date])(implicit messagingSystem: MessagingSystem) = Props(new ProcessSupervisor(process, date))
+
+  case object Done
+  case object Failed
+}
+
+class ProcessSupervisor(val process: StagedProcess, val startDate: Option[Date])(implicit val messagingSystem: MessagingSystem) extends Actor {
+  private val processManager: ActorRef = context.actorOf(Props(new ProcessManager(process, startDate)))
+
+  context.watch(processManager)
+
+  override def supervisorStrategy: SupervisorStrategy = OneForOneStrategy() {
+    case _ => Stop
+  }
+
+  override def receive: Receive = {
+    case ProcessSupervisor.Done => stopSystem(0)
+    case ProcessSupervisor.Failed => stopSystem(1)
```

Comment:
You could combine these two cases into `case ProcessSupervisor.Failed | Terminated(processManager) => stopSystem(1)`

Note that I didn't use backticks for `processManager` to avoid confusing the Markdown parser. Which actually leads me to the question: why use backticks here? The name is a valid Scala identifier name; backticks don't seem to be necessary.